File: ScriptGraph/BuiltInLua/CGGetAmgObjectNode.lua
local CGAmgObjectNode = CGAmgObjectNode or {}
CGAmgObjectNode.__index = CGAmgObjectNode

function CGAmgObjectNode.new()
    local self = setmetatable({}, CGAmgObjectNode)
    self.object = nil
    return self
end

function CGAmgObjectNode:getOutput(index)
    return self.object
end

return CGAmgObjectNode

File: ScriptGraph/BuiltInJS/CGGetAmgObjectNode.js
/**
 * @file CGGetAmgObjectNode.js
 * @author xuyuan
 * @date 2021/8/24
 * @brief CGGetAmgObjectNode.js
 * @copyright Copyright (c) 2021, ByteDance Inc, All Rights Reserved
 */

const Amaz = effect.Amaz;
const {BaseNode} = require('./BaseNode');

class CGGetAmgObjectNode extends BaseNode {
  constructor() {
    super();
    this.object = null;
  }

  getOutput(index) {
    return this.object;
  }
}

exports.CGGetAmgObjectNode = CGGetAmgObjectNode;
